server/src/main/scala/net/wiringbits/actions/GetPendingNotificationsAction.scala

```diff
@@ -0,0 +1,13 @@
+package net.wiringbits.actions
+
+import net.wiringbits.repositories.UserNotificationsRepository
+import net.wiringbits.repositories.models.UserNotification
+
+import javax.inject.Inject
+import scala.concurrent.Future
+
+class GetPendingNotificationsAction @Inject() (userNotificationsRepository: UserNotificationsRepository) {
```

Comment:
I like the idea to keep a single action per class, please use the `net.wiringbits.actions.internal` package instead to differentiate what relates to the end user APIs